farms/disc.py

```diff
@@ -70,7 +67,7 @@ def disc(ghi, sza, doy, pressure=101325, sza_lim=87):
 
     AM = (1. / (np.cos(np.radians(Ztemp))
                 + 0.15 * (np.power((93.885 - Ztemp), -1.253)))
-          * pressure / 101325)
+          * 100 * pressure / 101325)
```

Comment:
Whats the 100 * for?

Reply:
Ah, gotcha! F'ing confusing man. Okay so all inputs are in hPa/mbar but DISC assumes Pa, got it now.

Reply:
Exactly and previously disc just wanted Pa as input and did some stupid implicit scaling (rest did something similar) if it thought the units were bad but now its just all in mbar and its stated in the docstrings so if you input something wrong its your own fault haha

Reply:
Read the Docs people otherwise garbage in/garbage out!